integration-tests/src/test/java/be/orbinson/osgi/opentelemetry/it/OpenTelemetryIT.java

```diff
@@ -31,8 +39,25 @@ public void testOpenTelemetryBundlesActive() throws Exception {
         HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
 
         String responseBody = response.body();
-        assertEquals("Active", ((List<Map<String, Object>>) JsonPath.read(responseBody, "$.data[?(@.name=='opentelemetry-java')]")).get(0).get("state"));
-        assertEquals("Active", ((List<Map<String, Object>>) JsonPath.read(responseBody, "$.data[?(@.name=='opentelemetry-java-instrumentation')]")).get(0).get("state"));
+        assertEquals("Active", ((List<Map<String, Object>>) JsonPath.read(responseBody, "$.data[?(@.name=='opentelemetry-api')]")).get(0).get("state"));
```

Comment:
might use a helper method for readability

Reply:
done